Site/ASAB Maestro/Files/mongo/script/mongo-init.js

```diff
@@ -49,28 +39,47 @@ function main() {
 				continue;
 			}
 
-			if (!db.isMaster()) {
-				print("Not a master, continuing to the next Mongo node.");
+			const hello = db.hello()
```

Comment:
`hello` ? ... a better name pls - it will be visible in the MongoDB instance, so it must be relatable to this process.

Also consider deleting this DB at the end of this process.

Reply:
no, you don't understand. That's official. https://www.mongodb.com/docs/v6.3/reference/command/hello/#mongodb-dbcommand-dbcmd.hello

Reply:
ok, you mind the `hello` constant.

Reply:
I don't think that I create a new db when I enter the mongo shell...

Reply:
TODO: add comment